stream_alert/rule_processor/classifier.py

```diff
@@ -89,26 +89,25 @@ def refresh_record(self, new_record):
 
 class StreamClassifier(object):
     """Classify, map source, and parse a raw record into its declared type."""
-    def __init__(self, **kwargs):
-        self.config = kwargs['config']
+
+    def __init__(self, config):
+        self._config = config
         self._entity_log_sources = []
 
-    def map_source(self, payload):
+    @staticmethod
+    def extract_service_and_entity(raw_record):
```

Comment:
the `map` term is used throughout this method, I'd keep it in the method name unless we change the references below